### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by zach-nervana**
all of the commits here should be simple individual refactors

### CODE REVIEW

**Comment by zach-nervana** [comment]
@scttl are these CI test failures expected? The error says "Too long with no output (exceeded 30m0s)" in both cases even though in both cases all tests seem to have passed. Any idea what is going on here?

**Comment by scttl** [comment]
@zach-nervana yeah just noticed that too, definitely not expected.  The CI system is erroneously thinking the test is still running after it completes.  It may be a kubernetes pod cleanup or exit code issue.  I will start looking into this tomorrow.

**Comment by zach-nervana** [comment]
Cool thanks. Also, what do you think about changing that ci workflow so that all tests run without a review required. It might be nice for viewers to know that code already passed tests before reading through it.

**Comment by scttl** [comment]
the problem with doing that is that its fairly resource intensive to run each of the golden and trace tests against each environment on *every single commit* (not to mention its pretty wasteful eg. after a small doc update).  In addition to having them optionally kicked-off on PR's by reviewers that we have today, we have plans to run these nightly along with weekly full preset testing.

**Comment by scttl** [comment]
Looked into the CI failures and noticed that CircleCI had some reported workflow slowdowns and non processing issues which overlaps the time of these failures: https://status.circleci.com/  (they also had an outage the day prior).  In re-running these failed workflows today I haven't been able to reproduce so I think that's what was going on here.  Will monitor for further failures going forward.

**Comment by scttl** [comment]
FYI @zach-nervana I did some further digging into CI timeouts and found an issue related to a lack of token refresh for jobs exceeding 10 minutes.  Have a fix over on: #183 that should help here.  We still need to reset trace tests for those to pass, but all golden are passing now.

File: rl_coach/rollout_worker.py [hunk 1]
```diff
@@ -76,8 +76,10 @@ def rollout_worker(graph_manager, data_store, num_workers, task_parameters):
         chkpt_state_reader = CheckpointStateReader(checkpoint_dir, checkpoint_state_optional=False)
         last_checkpoint = 0
 
+        # this worker should play a fraction of the total playing steps per rollout
         act_steps = math.ceil((graph_manager.agent_params.algorithm.num_consecutive_playing_steps.num_steps)/num_workers)
 
+        # TODO: shouldn't this be improve_steps / num_consecutive_playing_steps?
```
Comment (Ajay191191) [inline rl_coach/rollout_worker.py]: Is this TODO still needed?

**Pull request merged**